dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/StoredCompany.kt

```diff
@@ -0,0 +1,13 @@
+package org.dataland.datalandbackend.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+/**
+ * Class for defining the fields needed to store a company in Dataland
+ * @param companyName as name for the company to be stored
+ * @param dataSets contains data identifiers for all data sets of this company and is initially an empty list
+ */
+data class StoredCompany(
```

Comment:
And again. Ist StoredCompany hier ein guter Name?

Reply:
Schon refactort. Also fix

Reply:
ok